nss-day-cohort-48/giffygram-hotbot
# ~minimode~

#### Changes Made

1. Added file `PostList.js` to `feed` directory.
2. Modified file `filename` to include `functionalityname` and functionality.
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout gqg-minimode-1
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > Verify that you are able to press the "Have a gif to submit button?" and the appropriate forms populate on top of the feed. Then, press cancel to collapse!

4. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
   > Checkout the comments at the bottom of PostList.js for a sneak peek at the window alerts soon to be in your futures.
